src/breadcrumb/index.css

```diff
@@ -27,39 +25,123 @@
 }
 
 .spectrum-Breadcrumbs-item {
-  display: block;
-  float: left;
+  display: inline-flex;
+  align-items: center;
+  justify-content: start;
 
-  position: relative;
+  box-sizing: border-box;
+  height: var(--spectrum-breadcrumb-list-height);
 
-  margin-right: var(--spectrum-breadcrumb-margin);
+  position: relative;
 
-  font-size: var(--spectrum-breadcrumb-text-size);
-  font-weight: var(--spectrum-breadcrumb-text-font-weight);
+  font-size: var(--spectrum-breadcrumb-item-text-size);
+  font-weight: var(--spectrum-breadcrumb-item-text-font-weight);
+  line-height: var(--spectrum-breadcrumb-item-text-line-height);
 
-  .spectrum-Breadcrumbs-itemLink {
-    cursor: default;
-    display: inline;
+  &:last-of-type {
+    font-weight: var(--spectrum-breadcrumb-title-text-font-weight);
 
-    &[href],
-    &[tabindex="0"] {
-      cursor: pointer;
+    .spectrum-Breadcrumbs-itemSeparator {
+      display: none;
     }
   }
 
-  &:last-of-type {
-    .spectrum-Breadcrumbs-itemSeparator {
-      display: none;
+  > .spectrum-ActionButton {
+    margin-right: var(--spectrum-breadcrumb-button-gap);
+  }
+}
+
+.spectrum-Breadcrumbs-itemLink {
+  cursor: default;
+  position: relative;
+
+  display: inline-flex;
+  align-items: center;
+  justify-content: start;
+
+  padding: 0 var(--spectrum-breadcrumb-item-padding-x);
+  height: var(--spectrum-breadcrumb-list-height);
+
+  /* Nate: Adding top padding breaks design for multiline */
+  /*padding-top: var(--spectrum-breadcrumb-multiline-list-padding-top);*/
+
+  /* Nate: Including margin-left makes things not match the design */
```

Comment:
@NateBaldwinDesign last we talked, I think we were good to remove these two bits, yeah?

Reply:
Correct, padding top would qualify as a platform specific override / omission since it renders as desired without the token.

As for margin-left, this was a duplicative token based on the way this was implemented and is not needed. This is good to be removed too.